pkg/volume/volume.go

```diff
@@ -132,11 +132,39 @@ type Attacher interface {
 	Attach() error
 }
 
```

Comment:
hmm.. The global vs bind concept is only applicable to attachable volumes. So how about I add `GetDeviceMountPath()` to `PhysicalAttacher` and leave `GetPath()` as is but expand the comment ?